dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -668,7 +668,20 @@ ssl.protocol.error.CWPKI0831E.useraction:Ensure that the configuration uses vali
 ssl.protocol.error.CWPKI0832E=CWPKI0832E: The {0} SSL/TLS protocol value that is provided cannot be used for configuring a list of SSL protocols values.
 ssl.protocol.error.CWPKI0832E.explanation:The protocol value is not valid in a protocol list grouping.
 ssl.protocol.error.CWPKI0832E.useraction:Ensure that the list of SSL/TLS protocols includes only values that are appropriate for a protocol list.
-# -------------------------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------------------------- 
 ssl.protocol.error.CWPKI0833E=CWPKI0833E: The {0} SSL/TLS configuration has an error that prevents creation of the SSL/TLS configuration. 
 ssl.protocol.error.CWPKI0833E.explanation:The SSL configuration attributes has an error that prevents SSLContext creation. 
 ssl.protocol.error.CWPKI0833E.useraction:Ensure that the SSL configuration attributes are correct. Review the logs for additional errors associated with SSL configuration attributes.
+# -------------------------------------------------------------------------------------------------
+ssl.config.error.CWPKI0834E=CWPKI0834E: The {0} SSL/TLS configuration cannot be set as the process default SSL configuration due to an error.
+ssl.config.error.CWPKI0834E.explanation:There is an error in the process default SSL configuration. 
+ssl.config.error.CWPKI0834E.useraction:Ensure the SSL configuration attributes are correct.  Review the logs for additional error associated with SSL configuration attributes.
```

Comment:
suggested edit

Ensure that the SSL configuration attributes are correct. Review the logs for errors that are  associated with SSL configuration attributes.